health.go

```diff
@@ -0,0 +1,18 @@
+package gapi
+
+type HealthResponse struct {
+	Code     string `json:"code,omitempty"`
+	Message  string `json:"message,omitempty"`
```

Comment:
Where are these two values coming from? They're not part of the API documentation and when I hit that endpoint locally I don't see those either.

Reply:
Yup good call, that's the structure of the error response and I forgot to remove them.